src/views/certifications/content/utils/read-local-file.ts

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs'
+import path from 'path'
+
+/**
+ * Read a local file from getStaticProps context,
+ * using `process.cwd()` to resolve the provided filePath.
+ *
+ * Return the file contents as a string.
+ */
+export function readLocalFile(filePath: string): string {
```

Comment:
💭 I could see this being a helpful util to lift up to `lib`

Reply:
👍 sweet, I dig it - have moved `read-local-file` and also `read-local-filepaths` into `src/lib` ✅